MSCL/source/mscl/MicroStrain/Inertial/InertialNode.cpp

```diff
@@ -999,4 +999,32 @@ namespace mscl
             Value::UINT8(0)
         });
     }
+
+    EventTriggerStatus InertialNode::getEventTriggerStatus(const std::vector<uint8> instances) const
+    {
+        std::vector<Value> specifier = { Value::UINT8(static_cast<uint8>(instances.size())) };
+
+        for (const uint8& instance : instances)
+        {
+            specifier.push_back(Value::UINT8(instance));
+        }
+
+        const MipFieldValues data = m_impl->get(MipTypes::CMD_EVENT_TRIGGER_STATUS, specifier);
+
+        const uint8 size = data[0].as_uint8();
+
+        EventTriggerStatus status(size);
+
+        for (int index = 0; index < size; ++index)
+        {
+            const int indexOffset = index * 2 + 1;
```

Comment:
Same point here as in [this comment](https://github.com/LORD-MicroStrain/MSCL/pull/223#discussion_r845270787).